modules/main_Projector/client/components/ProjectorActions.jsx

```diff
@@ -0,0 +1,47 @@
+import Presentations from '../../../main_SelectPresentation/globals/collections'
```

Comment:
Could you change this to 
'main_SelectPresentation/globals/collections'
please, thanks!